registrations/test_tasks.py

```diff
@@ -169,7 +169,9 @@ def test_opt_in(self):
         )
 
     @responses.activate
-    @mock.patch("registrations.tasks.group_send")
+    @mock.patch(
+        "registrations.tasks.validate_subscribe_jembi_app_registration." "send_webhook"
```

Comment:
We can make this one string

Reply:
I also don't think that this is necessary? Since we're calling the `send_webhook` function directly, we don't want to mock it out